### PR TITLE
Update requirements and setup script

### PR DESCRIPTION
**Pull request by svetozar-zikic**

### CODE REVIEW

File: scripts/set-aws-creds-file.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+mkdir -p ~/.aws
```
Comment (nijave) [inline scripts/set-aws-creds-file.sh]: Do we need AWS creds for anything? I don't think that should be needed since this is all going into public repos

Comment (svetozar-zikic) [reply scripts/set-aws-creds-file.sh]: We don't, I was thinking we needed it but we're adding a context with needed creds and not using aws to pull anything.

File: .circleci/config.yml [hunk 2]
```diff
@@ -139,10 +66,6 @@ jobs:
             . venv/bin/activate
```
Comment (nijave) [inline .circleci/config.yml]:
```suggestion
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -158,3 +81,32 @@ jobs:
           command: |
             . venv/bin/activate
```
Comment (nijave) [inline .circleci/config.yml]:
```suggestion
```

**Review by nijave — APPROVED** [review]

**Pull request merged**